Library/Homebrew/build_environment.rb

```diff
@@ -33,9 +33,15 @@ def std?
   module DSL
     extend T::Sig
 
+    def inherited(child)
```

Comment:
Can this get an inline comment? Not obvious what this is doing or why it's needed as-is.